plugins/generator-1.14.4/forge-1.14.4/procedures/_call_procedure.java.ftl

```diff
@@ -1,2 +0,0 @@
-<#include "procedures.java.ftl">

```

Comment:
Why did you delete this and some other files?

Reply:
I'm sorry I through they where unnecessary trash files.

Reply:
No, they are not. These are templates for important procedure blocks. Do not randomly remove things from the repository.

Reply:
It's done, I re-added them.